mautic/mautic
# Builder tiles debug

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | x
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Open email/page builder
2. Drag slot from slot types / section types onto builder
3. Observe visibility and style issues

#### Steps to test this PR:
1. Open email/page builder
2. Drag slot from slot types / section types onto builder
3. Observe visibility and style fixes
